### DISCUSSION TITLE
[General] <Unable to get listing on ZkSync Ecosystem page >

### DISCUSSION THREAD

**Discussion by Jerrybolt** [zkSync-Community-Hub/zksync-developers#115] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

Hello zkSync Team,

I'm Gerard, and I represent the team at FileMarket.xyz. Over the past couple of months, we've been really digging into zkSync, aiming to create a safe way to swap content using blockchain. While we're excited about this, we've hit a few snags as we try to fit into zkSync. Unfortunately, we haven't gotten any feedback or guidelines for how we could become a part of it.

We're enthusiastic contributors to this project, and we've been using zkSync from the start. But right now, we're feeling a bit discouraged by this situation. We're hoping that the zkSync team or Matter Labs could take a moment to read our message and give us some insights.

You can find me on Telegram: https://t.me/imonche

Best regards,
Gerard

### What is the specific issue or error you're encountering?

Listing

### Can you share the error messages or logs you're receiving, if any?

Listing

### Have you made any recent changes to the contract before encountering this issue?

Listing

### Are there any external libraries or contracts that your contract interacts with?

Listing

### Can you provide the relevant portions of your contract code where the issue is occurring?

Listing

### Have you tried to isolate the problem, and if so, what were the results?

Listing

### What steps have you already taken to try to resolve the issue?

Listing

**Comment by MexicanAce** [comment]
Hi @Jerrybolt 👋 

Have you submitted a [Typeform request](https://matterlabs.typeform.com/ecosystemportal?typeform-source=ecosystem.zksync.io) with your latest information?  If so, we can follow-up on that submission.

**Reply by Jerrybolt** [reply]
Hi @MexicanAce, yes we have submitted it, a month ago, I believe.

**Reply by MexicanAce** [reply, marked as answer]
Hi @Jerrybolt

After some follow-up/investigation, we've been working on a migration to [DappRadar](https://dappradar.com/) for a couple of months and we're in a transition phase (so that explains the slowness, apologies for that). This migration will complete in a couple of weeks and you can submit your project here 👉 [https://dappradar.com/dashboard/submit-dapp](https://dappradar.com/dashboard/submit-dapp)

As always, feel free to [join our Discord](https://join.zksync.dev/) and engage with our community there! ❤️

**Reply by rpuzakov-blip** [reply]
Hello! My name is Russ, I'm a Marketing Manager at Atomic Wallet, a non-custodial crypto wallet with over 2M+ monthly active users.

The submission form isn't working. Could you help with that? How can I add our project, Atomic Wallet, to your ecosystem page?

**Comment by grantimagen** [comment]
Dear Gerard,

Thank you for reaching out and expressing your interest in integrating FileMarket.xyz with zkSync. We appreciate your enthusiasm and commitment to contributing to the ecosystem.

To get started with integrating your platform with zkSync, we recommend reaching out directly to the zkSync team or Matter Labs through their official communication channels. You can also explore the developer documentation and resources provided by zkSync to gain insights into the integration process and best practices.

Additionally, consider joining zkSync community channels such as Discord or Telegram, where you can engage with other developers and enthusiasts to seek guidance and support for your integration efforts.

We look forward to seeing FileMarket.xyz become a part of the zkSync ecosystem and wish you success in your endeavors.

Best regards,
[Your Name]

**Comment by adambold** [comment]
As a representative of FileMarket.xyz, we are seeking guidance and support from the zkSync team or Matter Labs to integrate our content swapping platform with zkSync. We believe in the potential of zkSync and are eager to contribute to its ecosystem.

**Comment by shelleyalexander** [comment]
It seems like you're seeking guidance on integrating your project, FileMarket.xyz, with zkSync. I recommend reaching out directly to the zkSync team or Matter Labs through their official communication channels, such as their website contact form or community forums. They should be able to provide you with the necessary insights and support for integrating your project with zkSync.

**Comment by seimabirthcan** [comment]
Thank you for reaching out, Gerard! We appreciate your enthusiasm and interest in zkSync. Our team is dedicated to fostering collaboration and supporting projects like yours. Let's connect on Telegram to discuss how we can integrate FileMarket.xyz with zkSync and address any challenges you're facing. Looking forward to chatting with you!

**Comment by mooralfeapur** [comment]
The user is representing the team at FileMarket.xyz and is reaching out to the zkSync team for insights on how they could become a part of zkSync. The user has not provided any specific issue or error messages, nor have they provided any relevant portions of their contract code. No recent changes to the contract or external libraries/contracts are mentioned. No steps have been taken to isolate the problem, and no repo link is provided. No additional details are provided.
